### PR TITLE
create upcoming events view

### LINKED ISSUE #48: View Upcoming Events
Currently http://localhost:3000/events, displays all the events available. This has to be changed to display only the upcoming events. This can be done using the addition #37 .

We have the designs ready for displaying events, so we have to change the placeholder frontend to the actual designs (as in the image below). 

image
[Figma Link](https://www.figma.com/file/LEZGE9yXVag3LY8zHACAY5/Helping-Mamas-%2F-Spr2022?node-id=215%3A16692) has other scenario designs and components.

* Clicking on a calendar date should show the events of that particular date only.
* By default, no date should be chosen and all future events can be displayed.
* Dates with events available can be shown to have dots below them on the calendar.

* Note: Skip adding volunteer part that is in the design. (Can leave space for it in case we do need to add it in the future).

[This react plugin](https://demo.mobiscroll.com/react/calendar/event-hooks#) should be suitable for making the calendar, however feel free to use anything else if that is better suited.

### PR DESCRIPTION
**Pull request by jwu6492**
## PR Title/Tagline

What does this PR change and why?

### How to Test

1. Please add or build upon a testing card to the [Notion page](http://notion.so) and link it here

### Important Changes

- Database change / migration to run
- Environment config change
- Breaking API change

### Related Github Issues

- #48 
- Or, remember to link to an issue on the sidebar

### Related PRs

- #PRNUMBER

### Checklist

- [ ]  Database schema docs have been updated or are not necessary
- [ ]  Code follows design and style guidelines
- [ ]  Code is commented with doc blocks
- [ ]  Tests have been written and executed or are not necessary
- [ ]  Latest code has been rebased from base branch (usually `develop`)
- [ ]  Commits follow guidelines (concise, squashed, etc)
- [ ]  Github issues have been linked in relevant commits
- [ ]  Relevant reviewers (Senior Dev/EM/Designers) have been assigned to this PR

### CODE REVIEW

**Review by jackpalaia — APPROVED** [review]

File: src/screens/Events/User/EventManager.jsx [hunk 1]
```diff
@@ -5,27 +5,35 @@ import Icon from "../../../components/Icon";
...
     width: 100%;
     height: 100%;
     background: ${(props) => props.theme.grey9};
     padding-top: 1rem;
-    display: flex;
     flex-direction: column;
     align-items: center;
   `,
   HeaderContainer: styled.div`
     width: 95%;
```
Comment (abhigyank) [inline src/screens/Events/User/EventManager.jsx]: `width: 60%;`

File: src/screens/Events/User/EventManager.jsx [hunk 2]
```diff
@@ -5,27 +5,35 @@ import Icon from "../../../components/Icon";
...
     max-width: 80rem;
     display: flex;
-    justify-content: right;
+    justify-content: end;
     margin-bottom: 1rem;
+    padding-right: 5rem;
```
Comment (abhigyank) [inline src/screens/Events/User/EventManager.jsx]: replace:
```diff
-    margin-bottom: 1rem;
-    padding-right: 5rem;
+    margin: 0 auto;

```

File: src/screens/Events/User/EventManager.jsx [hunk 3]
```diff
@@ -34,6 +42,36 @@ const Styled = {
       box-shadow: none;
     }
   `,
+  Content: styled.div`
+    width: 100%;
```
Comment (abhigyank) [inline src/screens/Events/User/EventManager.jsx]: width: 60%

File: src/screens/Events/User/EventManager.jsx [hunk 4]
```diff
@@ -34,6 +42,36 @@ const Styled = {
...
+    height: 100%;
+    background: ${(props) => props.theme.grey9};
+    padding-top: 1rem;
+    display: flex;
+    flex-direction: row;
+    margin-left: 10rem;
```
Comment (abhigyank) [inline src/screens/Events/User/EventManager.jsx]: `margin: 0 auto;`

File: src/screens/Events/User/EventManager.jsx [hunk 5]
```diff
@@ -34,6 +42,36 @@ const Styled = {
...
+    align-items: start;
+  `,
+  EventContainer: styled.div`
+    width: 75%;
```
Comment (abhigyank) [inline src/screens/Events/User/EventManager.jsx]: width: 78%

File: src/screens/Events/User/EventTable.jsx [hunk 6]
```diff
@@ -11,59 +10,66 @@ const Styled = {
...
+                <Table.Slots>SLOTS</Table.Slots>
+                <Table.Register>
+                  <Styled.Button onClick={() => onRegister(event)}>
+                    <Icon color="grey3" name="add" />
+                    <span>Sign up</span>
+                  </Styled.Button>
+                </Table.Register>
+                <Table.Text>
+                  <Table.Volunteers>{event.volunteers.length}/{event.max_volunteers}          </Table.Volunteers>
+                  <Table.EventName>{event.name}</Table.EventName>
```
Comment (abhigyank) [inline src/screens/Events/User/EventTable.jsx]: #51 will update this to `event.title`, so we can change it here.

**Review by abhigyank — CHANGES_REQUESTED** [review]
Require some CSS changes to center divs.

**Comment by abhigyank** [comment]
Merged, thanks @jwu6492 :rocket:

**Pull request merged**